### ISSUE TITLE
Whats the newest version of GVR SDK I can use on Win32

### ISSUE THREAD

**Issue by SperoSophia** [googlevr/gvr-unity-sdk#600]
Since unity and gvr moved to unity3d pods implementation. my projects have failed.
I used GVR in the past with success but now.

I am building the iOS xcode project on windows, meaning that any cocoapods ( OSX exclusive ) cannot run on windows.

its the change from project to workspace.

Can i use 1.4 or 1.3 with unity 5.6 with native vr disabled in unity to fix your so called "improvement"

[Unity3d Forum Reference](https://forum.unity3d.com/threads/gvr-hack-for-unity-ios-build-pipeline-on-windows.472759/)

basically, how far back can should i go to a stable build that works on windows.

**Comment by bopangzz** [comment]
You should be able to use GVR Unity SDK 1.3  and unity 5.6 with native vr disabled for iOS cardbaord.

**Comment by Nicholas4417** [comment]
No there are not enough memory open space

**Comment by rusmaxham** [comment]
Where are you getting this error?  It is not possible to finish building a Unity iOS app from Windows. A Mac is required to finish the build with Xcode.  Are you getting this error on Windows or Mac?  Can you post more details about the error?

**Comment by fredsa** [comment]
The latest version of the SDK requires Cocoapods. Unfortunately, we're only able to support OSX + Xcode development for iOS.

We'll consider supporting alternatives to Cocoapods as a feature request.
